MinecraftClient/McClient.cs

```diff
@@ -2066,8 +2068,17 @@ public void OnLatencyUpdate(Guid uuid, int latency)
             string playerName = null;
             if (onlinePlayers.ContainsKey(uuid))
             {
+                Dictionary<int, Entity> keyValuePairs = GetEntities();

```

Comment:
(1/2) No need to call `GetEntities()` here, you can access `entities` directly since it's an internal field